src/app/static/src/js/components/reportLog/runningReportsList.vue

```diff
@@ -1,67 +1,45 @@
 <template>
-    <vue-typeahead-bootstrap
-        :data="sortedReports"
-        :serializer="e => e.name"
-        v-model="query"
-        showOnFocus
-        maxMatches="50"
-        placeholder="Choose a report"
-        @hit="$emit('update:key', $event.key)"
-    >
-        <template slot="append">
-            <button class="btn btn-outline-secondary" v-on:click.prevent="clear" v-if="query">
-                <x-icon/>
-            </button>
+    <v-select :options="sortedReports" label="name" :reduce="(label) => label.key" v-model="selectedKey"
+              :clearable="false" placeholder="Choose a report">
```

Comment:
Ability to clear selected report removed in favour of consistency with behaviour of workflow progress page. I can't really see a reason for it here anyway.

Reply:
I guess it made more sense in `vue-typeahead-bootstrap` which isn't as good at clearing the previous text out of the way when user re-focuses.